_posts/2015-04-13-Prepare-to-Jam.markdown

```diff
@@ -93,4 +93,4 @@ Here's a collection of useful links around the Jam and Testing in general:
  * [objc.io #15 - Testing](http://www.objc.io/issue-15/)
  * [iOS Test Coverage with Coveralls](http://list.her.sh/ios-test-coverage-with-coveralls/)
  * [Quality Coding Archives](http://qualitycoding.org/archives/)
- * [Dependency Injection, iOS and You](https://www.bignerdranch.com/blog/dependency-injection-ios/)
\ No newline at end of file
+ * [Dependency Injection, iOS and You](https://www.bignerdranch.com/blog/dependency-injection-ios/)
```

Comment:
I was editing the file through Github interface and I guess there was no empty line at the end and it was added automatically by Github.